.github/CODEOWNERS

```
@@ -1,13 +1,12 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
```

@jakirkham Could you please add owner for `python/pybind11` folder?

```suggestion
cpp/               @rapidsai/cucim-cpp-codeowners
python/pybind11    @rapidsai/cucim-cpp-codeowners
```